core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/type/HealthType.java

```diff
@@ -35,10 +36,20 @@
  * @since 0.0.2
  */
 public enum HealthType {
-  Unknown("Unknown"), Completed("Completed"), Failed("Failed"), Started("Started"), Stopped("Stopped"), Running("Running");
+  Unknown("Unknown", 0), Completed("Completed", -1), Failed("Failed", 0), Started("Started", -1), Stopped("Stopped", -1), Running("Running",
+      -1), Requested("Requested", 1);
```

Comment:
Sure. If added an extra property to `HealthType` where is describes if it can come from the sequencer and `StatusImpl` will throw if it's something crazy.